GoMeet/src/test/java/com/google/sps/VoteMeetingTimeServletTest.java

```diff
@@ -0,0 +1,284 @@
+package test.java.com.google.sps;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.ErrorMessages;
+import com.google.sps.servlets.VoteMeetingTimeServlet;
+import com.google.sps.data.MeetingTimeFields;
+import com.google.sps.data.ServletUtil;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.HashMap;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.Test;
+
+/**
+ * Tests for doPost of VoteMeetingTimeServlet
+ */
+@RunWith(JUnit4.class)
+public final class VoteMeetingTimeServletTest {
+  // Hardcoded data
+  private String MEETING_TIME_ID_VAL;
+  private final String DATETIME_VAL = "2021-01-20T16:33";
+  private final long VOTE_COUNT_VAL = 2;
+  // Note that HashSets will be stored as Lists in the Datastore, 
+  // but can be re-converted back to a set after fetch
+  private final HashSet<String> VOTERS_VAL = new HashSet<String>(Arrays.asList("John Smith", "Bob Citizen"));
+  private final String NEW_VOTER = "Mark Person";
+  private final String EXISTING_VOTER = "John Smith";
```

Comment:
Funny names :))

Reply:
I should have used John Citizen, which is what they like use on sample ID cards 😄